packages/lesswrong/lib/collections/revisions/collection.ts

```diff
@@ -48,6 +48,11 @@ Revisions.checkAccess = async (user: DbUser|null, revision: DbRevision, context:
   const { major: majorVersion } = extractVersionsFromSemver(revision.version)
   const collection = getCollection(collectionName);
   const documentId = revision.documentId;
+
+  if (!documentId) {
+    return false
```

Comment:
Annoyingly, we do actually have some revisions with missing `documentIds`.  If they don't belong to the current user attempting to access them (by their `userId`), then I don't know of a safer thing to do than to deny them permission to access the revision.  (I'm not actually sure what happens if you pass `null` into either a `loader.load` or a `collection.findOne` - with mongo I'm pretty sure the `findOne` would've returned a totally random document, which would be bad, but I doubt that's happening with postgres.)